### PR TITLE
SSH.koplugin: fix cant stop SSH server bug when pid file's stale

### PR DESCRIPTION
**Pull request by weijiuqiao**
See #9812.

- - -
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/10300)

### CODE REVIEW

**Review by NiLuJe — APPROVED** [review]

File: plugins/SSH.koplugin/main.lua [hunk 1]
```diff
@@ -95,6 +95,10 @@ function SSH:stop()
         timeout = 2,
     })
 
+    if self:isRunning() then
+        os.execute("rm /tmp/dropbear_koreader.pid")
```
Comment (Frenzie) [inline plugins/SSH.koplugin/main.lua:99]: Is there a particular reason to avoid `os.remove()`?

Comment (NiLuJe) [reply plugins/SSH.koplugin/main.lua:99]: Oh, duh', good point ;p.

Comment (NiLuJe) [reply plugins/SSH.koplugin/main.lua:99]: `os.execute` will fork a shell (it's literally a thin wrapper around `system`), while `os.remove` will call... `remove` (the libc function) ;p.

So, yeah, it's much more sensible ;).

Comment (weijiuqiao) [reply plugins/SSH.koplugin/main.lua]: Thanks! I've changed it accordingly.

**Pull request merged**